docker-compose.yaml

```diff
@@ -1,20 +1,14 @@
 version: '3.8'
 
 services:
-  php-7.2: &php
-    build: docker/php-7.2
+  php-7.4: &php
+    build: docker/php-7.4
     working_dir: /app
     volumes:
       - .:/app
     environment:
       PHP_IDE_CONFIG: serverName=php-cs-fixer
       PHP_CS_FIXER_ALLOW_XDEBUG: 1
-  php-7.3:
-    <<: *php
-    build: docker/php-7.3
-  php-7.4:
```

Comment:
we could keep 7.4 here I think?

Reply:
yes, good eye

Reply:
now it's on top of the file
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6203/files#diff-3fde9d1a396e140fefc7676e1bd237d67b6864552b6f45af1ebcc27bcd0bb6e9R4